### DISCUSSION TITLE
CP2K Release 2023.1

### DISCUSSION THREAD

**Discussion by oschuett** [cp2k/cp2k#2614] [Team Posts]
Hi Everybody,

following our tradition, we are planning another major release towards the end of December. 
So, please merge your changes and update the [version history](https://www.cp2k.org/version_history) in time.

**Comment by alazzaro** [comment]
I'm not sure if it makes sense to release 2022.3 or rather 2023.1, so that we get a fresh release at the beginning of the year... Concerning DBCSR, I'm planning to make a new version in the next 3 days.

**Comment by oschuett** [comment]
> 2023.1, so that we get a fresh release at the beginning of the year...

Yes, that's a good idea.

**Comment by oschuett** [comment]
The dashboard looks good. I don't see any blocking issues. I've drafted the release notes in the [version history](https://www.cp2k.org/version_history).
